runtime/lib/component/turen.js

```diff
@@ -599,13 +600,18 @@ Turen.prototype.toggleMute = function toggleMute (mute) {
  * Add an activation word.
  * @param {string} activationTxt
  * @param {string} activationPy
+ * @param {float} margin_index, 1-100
+ * @param {boolean} cloud_confirm, 1/0
  */
-Turen.prototype.addVtWord = function addVtWord (activationWord, activationPy) {
-  this.component.flora.post(VT_WORDS_ADD_WORD_CHANNEL, [
-    activationWord,
-    activationPy,
-    1
-  ])
+Turen.prototype.addVtWord = function addVtWord (activationWord, activationPy, margin_index, cloud_confirm) {
+  var msg = new Caps()
+  msg.writeString(activationWord)
+  msg.writeString(activationPy)
+  msg.writeInt32(1) // type of awake
+  msg.writeDouble(margin_index) //sensibility of awake
+  msg.writeInt32(cloud_confirm)
+  logger.info(`turen addVTWord, margin_index= ${margin_index} , cloud_confirm= ${cloud_confirm}`)
```

Comment:
This log should be more complete, because the fields `margin_index` and `cloud_confirm` belong to a vt word.

Reply:
yes, because turen api just receive some values params, could not reflect to vt-word data struct, so should remove the log here, and vt-word log info could get whole data struct values